docs/pvcscale.md

```diff
@@ -0,0 +1,179 @@
+# PVCScale Workload
+
+The pvcscale workload playbook is `workloads/pvcscale.yml` and will run the pvcscale workload on your cluster.
+
+PVCScale workload is designed to stress storage backend from the scale point of
+view. It will create desired number of pods and attach an PVC to each pod.
+During this test we measure time necessary for pods to be in `Ready/Ready` state
+and for all PVC to be in `Bound` state.
+
+For PVCScale test to work it is necessary that cluster is operational and there
+is working storage class which will be used to carve PVCs from. If storageclass
+is not present then this test will fail and will not work.
```

Comment:
will fail/will not work (one of them should do).

Reply:
fixed